logger.h

```diff
@@ -1,62 +1,123 @@
 /* Copyright © 2001-2014, Canal TP and/or its affiliates. All rights reserved.
-  
+
 This file is part of Navitia,
     the software to build cool stuff with public transport.
- 
+
 Hope you'll enjoy and contribute to this project,
     powered by Canal TP (www.canaltp.fr).
 Help us simplify mobility and open public transport:
     a non ending quest to the responsive locomotion way of traveling!
-  
+
 LICENCE: This program is free software; you can redistribute it and/or modify
 it under the terms of the GNU Affero General Public License as published by
 the Free Software Foundation, either version 3 of the License, or
 (at your option) any later version.
-   
+
 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 GNU Affero General Public License for more details.
-   
+
 You should have received a copy of the GNU Affero General Public License
 along with this program. If not, see <http://www.gnu.org/licenses/>.
-  
+
 Stay tuned using
-twitter @navitia 
+twitter @navitia
 IRC #navitia on freenode
 https://groups.google.com/d/forum/navitia
 www.navitia.io
 */
-
 #pragma once
+
 #include "conf.h"
+#include <boost/format.hpp>
+
 #include <log4cplus/logger.h>
 #include <log4cplus/configurator.h>
-#include <boost/format.hpp>
 #ifdef HAVE_LOGGINGMACROS_H
 #include <log4cplus/loggingmacros.h>
 #endif
 
-/** Crée une configuration par défaut pour le logger */
-inline void init_logger(){
+namespace navitia {
+
+// logger pattern example : [18-09-10 17:02:17,186] [INFO ] [NDC] - message file.cpp:248
+static const std::string console_pattern_ref = "[%D{%y-%m-%d %H:%M:%S,%q}] [%-5p] [%x] - %m %b:%L  %n";
+
+// syslog pattern example : Sep 11 14:16:11 hostname binary_name: [PID] [INFO] [NDC] - message file.cpp:261
```

Comment:
this comment doesn't seems to match the code

Reply:
It's because it represents the log in syslog

`Sep 11 14:16:11 hostname binary_name:` is the syslog part
`[PID] [INFO] [NDC] - message file.cpp:261` is the logger part

but after new changes it becomes

`Sep 11 16:50:09 par-lap06 ed2nav: [fr-idf] [32099] [INFO] [] - Building autocomplete data.cpp:381`